tensorflow_datasets/image_classification/__init__.py

```diff
@@ -96,3 +96,4 @@
 from tensorflow_datasets.image_classification.svhn import SvhnCropped
 from tensorflow_datasets.image_classification.uc_merced import UcMerced
 from tensorflow_datasets.image_classification.visual_domain_decathlon import VisualDomainDecathlon
+from tensorflow_datasets.image_classification.wake_vision import WakeVision
```

Comment:
This is a legacy folder, all new datasets should go into https://github.com/tensorflow/datasets/tree/master/tensorflow_datasets/datasets

Reply:
I have moved the wake_vision folder to the new datasets folder and removed the above line. From what I understand, I do not need to register my dataset in any __init__.py file for the new folder.

I believe the confusion stems from this [page](https://www.tensorflow.org/datasets/add_dataset#add_an_import_for_registration) which says:

>  For example, if you're contributing to tensorflow/datasets, add the module import to its subdirectory's __init__.py (e.g. [image/__init__.py](https://github.com/tensorflow/datasets/tree/master/tensorflow_datasets/image/__init__.py).

Perhaps this should be updated.